### PR TITLE
Adding artifacts + links from OCSF objects

### PR DESCRIPTION
**Pull request by ryantxu1**
Additions: 
- seeAlso links to ocsf object schema website
- Digital Fingerprint obj
- Entity/Host Group obj
- Scheduled Task obj
- Windows Registry Key/Value obj

_It struck me that digital artifacts don't have a space to provide great/lengthy detail (kb-article field in techniques)?_

### CODE REVIEW

**Comment by ryantxu1** [comment]
Still needs review

File: src/ontology/d3fend-protege.ttl [hunk 1]
```diff
@@ -9063,6 +9110,21 @@ Jobs can be scheduled in many different and sometimes creative ways through oper
         :Reference-PreventingExecutionOfTaskScheduledMalware_McAfeeLLC ;
     :synonym "Scheduled Job Execution" .
 
+:ScheduledTask a owl:Class ;
```
Comment (ryantxu1) [inline src/ontology/d3fend-protege.ttl:9113]: Create a Task Scheduler object

Comment (ryantxu1) [reply src/ontology/d3fend-protege.ttl:9113]: task scheduler creates scheduled task, updates(?)/reads task schedule, may manage scheduled task(?)

**Comment by netfl0** [comment]
let me know when ready for review.

**Comment by ryantxu1** [comment]
@netfl0 Ready for review

**Review by netfl0 — APPROVED** [review]

**Pull request merged**